lib/export/coldp.rb

```diff
@@ -47,11 +47,15 @@ def self.modified_by(updated_by_id, project_members)
       project_members[updated_by_id]
     end
 
+    def self.sanitize_remarks(remarks)
```

Comment:
@gdower This is OK here for now, but we should move it to /lib/utilities/strings.rb